### ISSUE TITLE
cannot find name 'iPromise'

### ISSUE THREAD

**Issue by stevo-knievo** [base33/Defined-Content#11]
Hi,
I just installed the package via nuget. Thanks for your work!
First I had to change the reference from `/// <reference path="scripts/typings/jquery/jquery.d.ts" />` to `/// <reference path="../jquery/jquery.d.ts"/>.`

After I fixed that I got `Cannot find name 'iPromise'.`

Cheers, Stefan

![ipromise errros](https://cloud.githubusercontent.com/assets/2980220/8896156/42858c66-33a8-11e5-86d9-a9b76e92741f.png)

**Comment by stevo-knievo** [comment]
I just fixed..

``````
/////////////////
//AssetService
/////////////////
interface assetService {
    load(pathArray: string[], scope: ng.IScope): resources.iPromise;
    loadCss(path: string, scope: ng.IScope, keyValue?: { [key: string]: any }, timeout?: number): resources.iPromise;
    loadJs(path: string, scope: ng.IScope, keyValue?: { [key: string]: any }, timeout?: number): resources.iPromise;
}```
``````

**Comment by DaveGreasley** [comment]
Hi @stevo-knievo,

Think you were probably looking for another repository: https://github.com/base33/DefinitelyTyped/

Defined-Content (this repo) is for an entirely different (and unreleased) Umbraco package. I can't see any commits in there yet but that seems to be the one you're after. You might want to check with Craig on twitter - https://twitter.com/craignoble1989.

I'm going to close this issue here for now.

**Comment by stevo-knievo** [comment]
opps - sorry!
I just followed the link External resources -> Issue tracker on the Umbraco project page.
 https://our.umbraco.org/projects/developer-tools/umbraco-7-typescript-definitions/

I would fix it but I can't find the umbraco typescript definitions here:
https://github.com/base33/DefinitelyTyped/

**Comment by DaveGreasley** [comment]
No worries! I'll get Craig to update the Umbraco page to point at the right repo tomorrow. 

If you send him a message on twitter he'll let you know where to post issues and where he's at with releasing the source.

**Comment by base33** [comment]
Hi @stevo-knievo I have fixed the issue in v1.1, which has now been released.

Thanks for reporting the issue.  The repo has now been created for the project.
https://github.com/base33/Umbraco-TypeScript-Definitions

Cheers
